pkg/log/logger.go

```diff
@@ -51,6 +53,15 @@ func NewLogger(level LogLevel) logr.Logger {
 	return NewLoggerTo(os.Stderr, level)
 }
 
+func NewLoggerWithRotation(level LogLevel, outputPath string, maxSize int, maxBackups int, maxAge int) logr.Logger {
+	klog.Info("logs will be stored in ", outputPath)
```

Comment:
This is just a constructor for logger, it should not log things.

Please log this just before those lines
```
core.SetLogger(core.NewLoggerWithRotation
```

also don't use `"k8s.io/klog/v2"` it's some random logger from K8S that we don't use. Just use `fmt.Printf` since we don't have our logger before we set it

Reply:
Right. Even I was thinking which library to use. Thanks, will make this changes.